apps/frontend/components/Report/Report.tsx

```diff
@@ -26,7 +27,7 @@ export const Report = ({
 
   if (isFloodArray(forms)) return <FloodReport forms={forms} />;
 
-  if (isDroughtArray(forms)) return <div>WIP DROUGHT</div>;
+  if (isDroughtArray(forms)) return <DroughtReport forms={forms} />;
```

Comment:
Linked to my previous comment - I feel like this is part of where we need to rethink the logical split to avoid duplication